src/devices/came_top432.c

```diff
@@ -0,0 +1,169 @@
+/** @file
+    Decode CAME remote control TOP-432EV, TOP-432NA, TOP-432EE.
+    This remote control is used for garage door and sliding gate. It transmits on 433.92 MHz (as it is written on the case), built since 2006.
```

Comment:
How do you know these exist since 2006? Maybe good to add a source here (if possible).